packages/core/src/parseAmplifyOutputs.ts

```diff
@@ -316,7 +316,13 @@ const providerNames: Record<string, OAuthProvider> = {
 };
 
 function getOAuthProviders(providers: string[] = []): OAuthProvider[] {
-	return providers.map(provider => providerNames[provider]);
+	return providers.reduce<OAuthProvider[]>((oAuthProviders, provider) => {
+		if (providerNames[provider] !== undefined) {
+			oAuthProviders.push(providerNames[provider]);
+		}
+
+		return oAuthProviders;
+	}, []);
```

Comment:
Can simplify this with `filter`:
```suggestion
	return providers.filter(provider => !!providerNames[provider]);
```

Reply:
slight note we would need map and filter in this case :P